Lab_5/CountriesRecommendation/CountriesRecommendation/CountrySizeParse.cs

```diff
@@ -0,0 +1,21 @@
+namespace CountriesRecommendation
+{
+    public class CountrySizeParse
+    {
+        public static CountrySize ParseCountrySize(string size) {
+            switch (size) {
```

Comment:
Еще одним трюком, который часто используется при сравнении строк - преобразование строки к нижнему регистру для сравнения. Вдруг пользователь решит написать `"Большая"`, в таком случае наш код сломается без этого преобразования.
```suggestion
            switch (size.ToLower()) {
```